isis/tests/FunctionalTestsMoccal.cpp

```diff
@@ -12,10 +12,10 @@ using namespace Isis;
 
 static QString APP_XML = FileName("$ISISROOT/bin/xml/moccal.xml").expanded();
 
-TEST_F(MgsMocCube, FunctionalTestMroMoccalDefault) {
+TEST_F(MgsMocCube, FunctionalTestMromoccalDefault) {
```

Comment:
Not sure how this got through the first time. "moccal" is an MGS app not an MRO. On the other hand, the app name is "moccal" not "mgsmoccal", shouldn't this just be "FunctionalTestMoccalDefault" ?